src/array19.lib/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, CMake
+from conan.tools.cmake import CMakeToolchain
+import os
+
+class Array19Conan(ConanFile):
+    name = "array19"
+    version = "1.0"
+    
+    # Optional metadata
+    license = "MIT License"
+    author = "Hicknhack Software"
+    url = "https://github.com/basicpp17/co-cpp19"
+    description = "C++17/20 Library with the fastest runtime and compile times"
+    topics = ("algorithm", "container", "common", "utility")
+    generators = "cmake_find_package", "cmake"
+    # Binary configuration
```

Comment:
array19 is header only. Does it work without these "Binary configuration"?

Reply:
is removed